daprdocs/content/en/operations/components/pluggable-components-registration.md

```diff
@@ -49,49 +49,43 @@ Both your component and the Unix Socket must be running before Dapr starts.
 
 By default, Dapr looks for [Unix Domain Socket][uds] files in the folder in `/tmp/dapr-components-sockets`.
 
-Filenames in this folder are significant for component registration. They must be formed by appending the component's name with a file extension of your choice, more commonly `.sock`. For example, the filename `my-component.sock` is a valid UDS file name for a component named `my-component`.
+Filenames in this folder are significant for component registration. They must be formed by appending the component's **name** with a file extension of your choice, more commonly `.sock`. For example, the filename `my-component.sock` is a valid UDS file name for a component named `my-component`.
 
 Since you are running Dapr in the same host as the component, verify this folder and the files within it are accessible and writable by both your component and Dapr.
 
-## Define the component
+### Component discovery and multiplexing
 
-Define your component using a [component spec]({{< ref component-schema.md >}}). Your component's `type` is derived from the socket name, without the file extension.
+A pluggable component accessible through a [Unix Domain Socket][uds] can host multiple distinct bulding blocks. During the components initial discovery process, Dapr will use reflection to enumerate all building blocks behind a UDS. The `my-component` pluggable component from the example above could contain both a state store (`state`) and a Pub/Sub (`pubsub`) building-blocks.
 
-Save the component YAML file in the resources-path, replacing:
+The most common use-case is for a 1:1 mapping between pluggable components and building blocks they expose. That said, having the possibility of consolidating multiple building blocks under the same pluggable components might be healpful, specially to ease DevOps burden at the expense of fault tolerance and security. Please weight the pros and cons and, if in doubt, stick to a 1:1 approach.
```

Comment:
```suggestion
Typically a pluggable component implements a single component interface for packaging and deployment. However a pluggable component can have multiple component interfaces implemented  to ease the deployment burden at the expense of having dependencies and potentially security. The recommendation for isolation, fault tolerance and security is a single component interface implementation for each pluggable components, however this is simply a guideline. 
```

Reply:
Suggest "component API" instead of "component interface" to be consistent with above text.  (I don't have a strong preference for either one, but consistent phrasing would be best.)